src/credentials/attestation_verifier/DeviceAttestationVerifier.cpp

```diff
@@ -111,5 +113,40 @@ void SetDeviceAttestationVerifier(DeviceAttestationVerifier * verifier)
     gDacVerifier = verifier;
 }
 
+static inline ByteSpan DupByteSpanHelper(const ByteSpan & span_to_dup)
+{
+    uint8_t * bytes = (uint8_t *) chip::Platform::MemoryAlloc(span_to_dup.size());
```

Comment:
This is risky of a memory allocation scheme. Suggest more explicitly maintaining copies

Reply:
Could you elaborate how you want the copies explicitly maintained?

Reply:
This has been replaced with Boris's suggestion of using ScopedMemoryBufferWithSize and UniquePtr.